packages/fxa-auth-server/lib/routes/support.js

```diff
@@ -68,7 +67,7 @@ module.exports = (log, db, config, customs, zendeskClient) => {
         const { uid, email } = await handleAuth(request.auth, true);
         const { location } = request.app.geo;
         await customs.check(request, email, 'supportRequest');
-        let subject = `${request.payload.plan}`;
+        let subject = `${request.payload.productName}`;
```

Comment:
Why is this code in this PR?  Doesn't seem to be fluent related.

Reply:
This was updated because `plan` had been removed from the payload, and this had not been corrected.

Reply:
Sure, but this PR is for Fluent changes, and that code fixes https://github.com/mozilla/fxa/issues/4889.

Reply:
Yes, as stated in #4889, this PR fixes that issue.

Reply:
It was fixed here, but can be pulled out. I was afraid that the tests would fail upon trying to open a PR if that had not been fixed. There were errors that resulted from the plan -> product name usage that were not caught because the tests do not run if the PR includes a branch from a forked repo. I had thought that the tests might fail in this PR, since I decided to move my work to `mozilla/fxa`, so I went ahead and fixed it.

Reply:
Huh.  I guess then I don't understand why tests would fail now without this change when they didn't fail in PR #4694.  

A separate smaller PR is nicer because then the changelog(s) would make more sense (although I'm not even sure anyone actually read them), and the unit of change can be cherry-picked or reverted easily.

(I already r+ so feel free to merge when you thing the PR is ready. 😀)